src/library_dylink.js

```diff
@@ -33,6 +33,9 @@ var LibraryDylink = {
       sym = createInvokeFunction(symName.split('_')[1]);
     }
 
+    if(!sym && symName.startsWith("__cxa_find_matching_catch")){
```

Comment:
```suggestion
    if (!sym && symName.startsWith("__cxa_find_matching_catch")) {
```

Reply:
Nice!  lgtm assuming we can write a test for it.